charts/workflow/values.yaml

```diff
@@ -122,13 +122,12 @@ database:
     host: "database host"
     port: "database port"
 
-logger:
-  redis:
-    # Configure the following ONLY if using an off-cluster Redis instance for logger
-    db: "0"
-    host: "redis host"
-    port: "redis port"
-    password: "redis password" # "" == no password
+redis:
+  # Configure the following ONLY if using an off-cluster Redis instance for logger
+  db: "0"
+  host: "redis host"
+  port: "redis port"
+  password: "redis password" # "" == no password
```

Comment:
Cannot reproduce it at the moment. I think the reason of my issue was me changing redis from on-cluster to off-cluster and vice versa, probably some artifacts. Just set up 2.10 cluster, it runs with correct one-work `requirepass`.